python/tank/bootstrap/cached_configuration.py

```diff
@@ -278,5 +279,6 @@ def _cleanup_backup_folders(self, config_backup_folder_path, core_backup_folder_
             if path:
                 try:
                     filesystem.safe_delete_folder(path)
+                    log.debug("Deleted backup folder: %s" % path)
```

Comment:
You can do the following instead:

```
log.debug("Deleted backup folder: %s", path)
```

The logger has a mechanism where you can defer string interpolation until when the message is being printed. That means if debug logging is turned off, we don't spend time on it. It's a SUPER small thing for a single debug message, but in aggregate, across our entire codebase, it makes a difference in performance.